Vagrantfile

```diff
@@ -0,0 +1,68 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+hostname      = 'st2-packages'
+st2user       = ENV['ST2USER'] ? ENV['ST2USER']: 'st2admin'
+st2passwd     = ENV['ST2PASSWORD'] ? ENV['ST2PASSWORD'] : 'Ch@ngeMe'
+box           = ENV['BOX'] ? ENV['BOX'] : 'centos/7'
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+VIRTUAL_MACHINES = {
+  :ubuntu14 => {
+    :hostname => 'ubuntu14',
+    :box => 'ubuntu/trusty64',
+    :target => 'trusty',
+  },
+  :ubuntu16 => {
+    :hostname => 'ubuntu16',
+    :box => 'ubuntu/xenial64',
+    :target => 'xenial',
+  },
+  :centos6 => {
+    :hostname => 'centos6',
+    :box => 'centos/6',
+    :target => 'el6',
+  },
+  :centos7 => {
```

Comment:
`vagrant up el7`, same as we define them in https://github.com/StackStorm/st2-packages/blob/master/circle.yml#L19
don't need `:target`.

Additionally, it might be good to include in `hostname` something specific with packages, ex: `st2-packages-el7`, otherwise it could be confused with https://github.com/StackStorm/ansible-st2/blob/master/Vagrantfile#L19